src/Generator/Commands/TestCases/CliTestCaseGenerator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Apiato\Core\Generator\Commands\TestCases;
+
+use Apiato\Core\Generator\FileGeneratorCommand;
+
+class CliTestCaseGenerator extends FileGeneratorCommand
+{
+    protected string|null $fileName = 'CliTestCase';
+
+    public static function getCommandName(): string
+    {
+        return 'apiato:generate:testcase:functional:cli';
```

Comment:
`make`

Reply:
Done in de2ba4c2e6127447ab7e4eff8306eda135f73cfa